cps/fb2.py

```diff
@@ -4,8 +4,10 @@
 from lxml import etree
 import os
 import uploader
-import StringIO
-
+try:
+    import StringIO
+except:
```

Comment:
To catch any exception use the ```except Exception```, instead of ```except BaseException``` or just ```except```.
Indicate the most specific exception type in ```except``` block.
For example:
```python
# Bad
try:
    mapping[key]
except Exception:
    ...

# Better
try:
    mapping[key]
except KeyError:
    ...
```